x-pack/plugins/lens/public/index.scss

```diff
@@ -1,12 +1,13 @@
 // Import the EUI global scope so we can use EUI constants
-@import 'src/legacy/ui/public/styles/_styling_constants';
+@import '@elastic/eui/src/global_styling/variables/index';
```

Comment:
:information_source: I replaced the import of `_styling_constants` in the legacy world by importing the relevant two files we need from EUI directly.